### PR TITLE
Fix create graphql-modules example

### PR DESCRIPTION
**Pull request by alaarihan**
When I ran build it failed so I fixed these two small issues to make the build work.

### CODE REVIEW

File: packages/create/examples/graphql-modules/src/app/common/common.module.ts [hunk 1]
```diff
@@ -3,6 +3,6 @@ import { PrismaProvider } from './Prisma.provider';
 import { sdlInputs } from '@paljs/plugins';
 
 export const CommonModule = new GraphQLModule({
-  typeDefs: sdlInputs(),
+  typeDefs: sdlInputs({}),
```
Comment (AhmedElywa) [inline packages/create/examples/graphql-modules/src/app/common/common.module.ts:6]: Fix this issue not here but in the `sdlInputs` function place to make args optional 
 here https://github.com/paljs/prisma-tools/blob/f3f35b57b5c9fac4cfd4b4361feddca5897ef04a/packages/plugins/src/sdlInputs.ts#L103

**Review by alaarihan — COMMENTED** [review]
Fixed

**Comment by AhmedElywa** [comment]
thanks for your work

**Comment by alaarihan** [comment]
It's my pleasure brother :) thank you!

**Pull request merged**